packages/webdriver/src/bidi/core.ts

```diff
@@ -1,5 +1,7 @@
 import logger from '@wdio/logger'
 import type { ClientOptions, RawData, WebSocket } from 'ws'
+import { isIP } from 'node:net'
+import dns from 'node:dns'
```

Comment:
Since we only use the promise primitives anyway.
```suggestion
import dns from 'node:dns/promises'
```

Reply:
changed